internal/services/kusto/kusto_attached_database_configuration_resource.go

```diff
@@ -73,8 +73,7 @@ func resourceKustoAttachedDatabaseConfiguration() *pluginsdk.Resource {
 				ValidateFunc: validation.Any(validate.DatabaseName, validation.StringInSlice([]string{"*"}, false)),
 			},
 
-			// TODO: this should become `cluster_id` in 4.0
-			"cluster_resource_id": {
+			"cluster_id": {
```

Comment:
NOTE: this is a WIP, I still need to go through the full property-rename procedure